packages/angular/cli/commands/doc-impl.ts

```diff
@@ -14,13 +14,18 @@ const opn = require('opn');
 
 export class DocCommand extends Command<DocCommandSchema> {
   public async run(options: DocCommandSchema & Arguments) {
-    let searchUrl = `https://angular.io/api?query=${options.keyword}`;
-    if (options.search) {
-      searchUrl = `https://www.google.com/search?q=site%3Aangular.io+${options.keyword}`;
-    }
+    if (options.keyword) {
+      let searchUrl = `https://angular.io/api?query=${options.keyword}`;
+      if (options.search) {
+        searchUrl = `https://www.google.com/search?q=site%3Aangular.io+${options.keyword}`;
+      }
 
-    return opn(searchUrl, {
+      return opn(searchUrl, {
```

Comment:
The indentation doesn't look right. `wait: false` needs two more tabs?

Reply:
😱